GameOfRunes/GameOfRunes/GameEngine/Entities/PowerUps/HellfirePowerUpEntity.swift

```diff
@@ -10,10 +10,8 @@ import SpriteKit
 import GameplayKit
 
 /** Entity to represent the Hellfire Power Up */
-class HellfirePowerUpEntity: Entity, PowerUp {
-    var powerUpType: PowerUpType {
-        .hellfire
-    }
+class HellfirePowerUpEntity: Entity {
+    var powerUp: PowerUp = HellfirePowerUp.shared
```

Comment:
can remove this actually, since we don't need this here

Reply:
why not?

Reply:
we need it when it is accessed from outside, like from other classes